site/_docs/reference.md

```diff
@@ -2658,6 +2658,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | s | ARRAY_REPEAT(element, count)                   | Returns the array containing element count times.
 | b | ARRAY_REVERSE(array)                           | Reverses elements of *array*
 | s | ARRAY_SIZE(array)                              | Synonym for `CARDINALITY`
+| s | SORT_ARRAY(array [, ascendingOrder])           | Sorts the input array in ascending or descending order according to the natural ordering of the array elements. Null elements will be placed at the beginning of the returned array in ascending order or at the end of the returned array in descending order
```

Comment:
array should be *array*, Also I don't know if it's clear from this what the default behavior is if `ascendingOrder` is not specified. I think this could be made a little more clear.

Reply:
add in the doc @tanclary